doc/whats_new/v0.21.rst

```diff
@@ -413,6 +413,10 @@ Support for Python 3.4 and below has been officially dropped.
 - |API| :class:`pipeline.Pipeline` now supports using ``'passthrough'`` as a
   transformer. :issue:`11144` by :user:`Thomas Fan <thomasjpfan>`.
 
+- |Enhancement| :class:`pipeline.Pipeline` implements a func len,
```

Comment:
... implements ``__len__`` and therefore ``len(pipeline)`` returns the number of steps in the pipeline.

Reply:
ok modifying message.